tests/conftest.py

```diff
@@ -60,3 +64,8 @@ def ingest_pipeline():
 @pytest.fixture(scope='function')
 def target_api_config():
     return TargetAPIConfig(KIDS_FIRST_CONFIG)
+
+
+@pytest.fixture(scope='function')
```

Comment:
Make a reusable fixture for transform stage because now we need to use it across multiple tests